### PR TITLE
[schemes] Add dynamic color scheme MDCColorSchemeDefaultsMaterial201907.

### PR DESCRIPTION
**Pull request by fluxminds**
Part of b/137088446.

### CODE REVIEW

File: components/schemes/Color/src/MDCSemanticColorScheme.m [hunk 1]
```diff
@@ -69,6 +84,18 @@ - (instancetype)initWithDefaults:(MDCColorSchemeDefaults)defaults {
...
         break;
+      case MDCColorSchemeDefaultsMaterial201907:
+        _primaryColor = DynamicColor(ColorFromRGB(0x6200EE), ColorFromRGB(0xBB86FC));
+        _primaryColorVariant = DynamicColor(ColorFromRGB(0x3700B3), ColorFromRGB(0x3700B3));
+        _secondaryColor = DynamicColor(ColorFromRGB(0x03DAC6), ColorFromRGB(0x03DAC6));
+        _errorColor = DynamicColor(ColorFromRGB(0xB00020), ColorFromRGB(0xCF6679));
+        _surfaceColor = DynamicColor(ColorFromRGB(0xFFFFFF), ColorFromRGB(0x121212));
+        _backgroundColor = DynamicColor(ColorFromRGB(0xFFFFFF), ColorFromRGB(0x121212));
+        _onPrimaryColor = DynamicColor(ColorFromRGB(0xFFFFFF), ColorFromRGB(0x000000));
+        _onSecondaryColor = DynamicColor(ColorFromRGB(0x000000), ColorFromRGB(0x000000));
```
Comment (bryanoltman) [inline components/schemes/Color/src/MDCSemanticColorScheme.m:99]: Should this be black in both cases?

File: components/schemes/Color/src/MDCSemanticColorScheme.m [hunk 2]
```diff
@@ -69,6 +84,18 @@ - (instancetype)initWithDefaults:(MDCColorSchemeDefaults)defaults {
         _onSurfaceColor = ColorFromRGB(0xFFFFFF);
         _onBackgroundColor = ColorFromRGB(0xFFFFFF);
         break;
+      case MDCColorSchemeDefaultsMaterial201907:
+        _primaryColor = DynamicColor(ColorFromRGB(0x6200EE), ColorFromRGB(0xBB86FC));
+        _primaryColorVariant = DynamicColor(ColorFromRGB(0x3700B3), ColorFromRGB(0x3700B3));
```
Comment (bryanoltman) [inline components/schemes/Color/src/MDCSemanticColorScheme.m:93]: Should this be the same in both cases?

File: components/schemes/Color/src/MDCSemanticColorScheme.m (hunk 1 above)
Comment (fluxminds) [reply components/schemes/Color/src/MDCSemanticColorScheme.m:99]: From https://material.io/design/color/dark-theme.html#ui-application, it uses black for both cases.

File: components/schemes/Color/src/MDCSemanticColorScheme.m (hunk 2 above)
Comment (fluxminds) [reply components/schemes/Color/src/MDCSemanticColorScheme.m:93]: Same in this one. Reference: https://material.io/design/color/dark-theme.html#ui-application

**Review by bryanoltman — APPROVED** [review]

**Pull request merged**